Before: 05239fd9b0b89a279110feaf2ae8b9effc4045939665a1999f75ee8e38db830d
After: ebca4347a70ae1bf5753b876ef4d781e2fdfeddc9b28ab9b535c79798e20b6d3
Patch: 9663392192ea9f415a3d70762b234fd94fde5f99a40f11a20835dd196f4d0d09
--- test/bootstrap/all.rb	2012-07-22 20:44:49.515582345 +1000
+++ /dev/null	2012-07-22 20:29:06.375709578 +1000
@@ -1,2 +0,0 @@
-# encoding: utf-8
-APP_DIR = File.expand_path('../../..',__FILE__)
--- test/bootstrap/spec.rb	2012-07-22 20:44:49.515582345 +1000
+++ /dev/null	2012-07-22 20:29:06.375709578 +1000
@@ -1,3 +0,0 @@
-# encoding: utf-8
-require_relative 'all'
-
--- test/bootstrap/unit.rb	2012-07-22 20:44:49.515582345 +1000
+++ /dev/null	2012-07-22 20:29:06.375709578 +1000
@@ -1,3 +0,0 @@
-# encoding: utf-8
-require_relative 'all'
-
--- test/spec/%src%_spec.rb.tt	2012-07-22 20:44:49.515582345 +1000
+++ /dev/null	2012-07-22 20:29:06.375709578 +1000
@@ -1,7 +0,0 @@
-# encoding: utf-8
-require_relative '<%= bootstrap_dir %>/spec'
-require '<%= src %>'
-
-describe <%= subject %> do
-  # TODO
-end
--- test/unit/%src%_test.rb.tt	2012-07-22 20:44:49.515582345 +1000
+++ test/unit/%src%_test.rb.tt	2012-07-22 21:27:39.120374754 +1000
@@ -1,5 +1,5 @@
 # encoding: utf-8
-require_relative '<%= bootstrap_dir %>/unit'
+require_relative '<%= bootstrap %>'
 require '<%= src %>'
 
 class <%= testcase_name %>Test < MiniTest::Unit::TestCase
